PowerEditor/src/MISC/PluginsManager/PluginsManager.cpp

```diff
@@ -529,7 +529,7 @@ bool PluginsManager::relayPluginMessages(UINT Message, WPARAM wParam, LPARAM lPa
 				{
 					TCHAR funcInfo[128];

```

Comment:
I changed the one buffer because the format string itself was bigger than the buffer. These two I left alone since they were fine.